lib/index.js

```diff
@@ -0,0 +1,2 @@
+//foods.html
+require('./ajax-requests/all_foods.js')
```

Comment:
So does everything kind of flow through the ajax-requests file? How does webpack know about the other files?